ci/download_dependencies.sh

```diff
@@ -0,0 +1,135 @@
+#!/bin/bash
+set -euxo pipefail
+
+source "${WORKDIR}/ci/archs.sh"
+
+temp_dir="${WORKDIR}/bin/deps/releases"
+lib_root="${WORKDIR}/bin/deps/lib"
+
+libtelio_artifact_url="${LIBTELIO_RELEASES_URL}/${LIBTELIO_VERSION}/downloads/linux.zip"
+libtelio_zipfile="${temp_dir}/libtelio-${LIBTELIO_VERSION}.zip"
+libtelio_dst="${temp_dir}/libtelio-${LIBTELIO_VERSION}"
+
+libdrop_artifact_url="${LIBDROP_RELEASES_URL}/${LIBDROP_VERSION}/downloads/linux.zip"
+libdrop_zipfile="${temp_dir}/libdrop-${LIBDROP_VERSION}.zip"
+libdrop_dst="${temp_dir}/libdrop-${LIBDROP_VERSION}"
+
+libmoose_nordvpnapp_artifact_url="${LIBMOOSE_RELEASES_URL}/${LIBMOOSE_NORDVPNAPP_VERSION}/downloads/linux.zip"
+libmoose_nordvpnapp_zipfile="${temp_dir}/libmoose-nordvpnapp-${LIBMOOSE_NORDVPNAPP_VERSION}.zip"
+libmoose_nordvpnapp_dst="${temp_dir}/libmoose-nordvpnapp-${LIBMOOSE_NORDVPNAPP_VERSION}"
+
+libmoose_worker_artifact_url="${LIBMOOSE_RELEASES_URL}/${LIBMOOSE_WORKER_VERSION}/downloads/linux.zip"
+libmoose_worker_zipfile="${temp_dir}/libmoose-worker-${LIBMOOSE_WORKER_VERSION}.zip"
+libmoose_worker_dst="${temp_dir}/libmoose-worker-${LIBMOOSE_WORKER_VERSION}"
+
+mkdir -p "${temp_dir}"
+
+function fetch_gitlab_release() {
```

Comment:
naming: As artifacts downloaded can be any uploaded artifacts, not just releases, this should be named `feetch_gitlab_artifact`. `release_url` needs to be renamed as well

Reply:
Done :+1: